test_xenon.py

```diff
@@ -179,5 +184,299 @@ def test_api(self):
                           'message': 'Resource creation started'})
 
 
+class TestCustomConfigParserGetStr(unittest.TestCase):
+    '''Test class for class CustomConfigParser - getstr method.'''
+
+    @staticmethod
+    def get_configuration(text):
+        '''Get CustomConfigParser object with loaded text.'''
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            pyproject_path = tmp_dir + '/pyproject.toml'
+            with open(pyproject_path, "w") as toml_file:
+                toml_file.write(text)
+
+            configuration = xenon.CustomConfigParser()
+            configuration.read(pyproject_path)
+
+        return configuration
+
+    def test_missing_section_case(self):
+        '''Test missing section case.'''
+        configuration = self.get_configuration(
+            '[tool.xenon]\n'
+            'path = ["path_1", "path_2", "path_3"]\n'
+            'max_average = "A"\n'
+            'max_average-num = 1.2\n')
```

Comment:
Shouldn't this one be `max_average_num`?

Reply:
Corrected. Thank you.